portal/code/cx-portal/src/index.tsx

```diff
@@ -1,19 +1,19 @@
 import React from 'react'
 import ReactDOM from 'react-dom'
 import { Provider } from 'react-redux'
-import { store } from './state/store'
-import Router from './components/Router'
-import UserService from './services/UserService'
-import I18nService from './services/I18nService'
+import { store } from 'state/store'
+import Router from 'components/Router'
+import I18nService from 'services/I18nService'
 
+//initialise translation library
 I18nService.init()
-UserService.init(() => {
-  ReactDOM.render(
-    <React.StrictMode>
-      <Provider store={store}>
-        <Router />
-      </Provider>
-    </React.StrictMode>,
-    document.getElementById('app')
-  )
-})
+
```

Comment:
The problem was that we had
`UserService.init(() => ReactDOM.render(...))`
instead of
`UserService.init(() => { ReactDOM.render(...) } )`
in the first line the pointer function is using the return value of the render function. So we simply wrap it in {} to create a function that doesn't return any vallue.